src/graphql/resolveConnection.js

```diff
@@ -51,14 +57,33 @@ const resolveConnection = (
       return sql
     })
 
+    // Gets the condition for filtering our result set using a cursor.
+    const getCursorCondition = (cursor, operator) => {
+      const sql = new SQLBuilder()
+
+      const cursorCompareLHS = `(${
+        [orderBy.name, ...primaryKeyNoOrderBy.map(({ name }) => name)]
+          .map(name => `"${name}"`)
+          .join(', ')
+      })`
+
+      const cursorCompareRHS = `(${
+        [null, ...primaryKeyNoOrderBy].map((x, i) => `$${i + 1}`).join(', ')
```

Comment:
Here we really just care about the length so we can generate the right number of placeholders. So `Array(1 + primaryKeyNoOrderBy.length).fill(null)` is better. Thanks for the catch!